package.json

```diff
@@ -78,7 +78,7 @@
         "lodash-es": "^4.17.21",
         "rxjs": "^6.6.3",
         "shaka-player": "3.1.0",
-        "@azure/video-analyzer-player": "1.0.3",
+        "@azure/video-analyzer-player": "1.0.4",
```

Comment:
I just realized, you forgot to add your package-lock.json changes, you should do so.

Reply:
I added the file but somehow it doesn't show up here

Reply:
If you see file changed it is there but hidden.